var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -117,6 +119,10 @@ class Vtk(CMakePackage):
     depends_on("gl@3.2:", when="+opengl2")
     depends_on("gl@1.2:", when="~opengl2")
 
+    depends_on("xz")
+    patch("vtk_find_liblzma.patch", when="@:9.0")
+    patch("vtk_movie_link_ogg.patch")
```

Comment:
```
==> Installing vtk-9.2.6-3antmzo7jyw2rvf6uxeyd2isrgl7mbjz [335/382]
==> No binary for vtk-9.2.6-3antmzo7jyw2rvf6uxeyd2isrgl7mbjz found: installing from source
==> Using cached archive: /Users/Adam/spack/var/spack/cache/_source-cache/archive/06/06fc8d49c4e56f498c40fcb38a563ed8d4ec31358d0101e8988f0bb4d539dd12.tar.gz
File to patch: 
No file found--skip this patch? [y] 
1 out of 1 hunks ignored--saving rejects to Oops.rej
==> Patch /Users/Adam/spack/var/spack/repos/builtin/packages/vtk/vtk_movie_link_ogg.patch failed.
==> Error: ProcessError: Command exited with status 1:
    '/usr/bin/patch' '-s' '-p' '1' '-i' '/Users/Adam/spack/var/spack/repos/builtin/packages/vtk/vtk_movie_link_ogg.patch' '-d' '.'
```
Should this be restricted to a subset of versions?

Reply:
Same for `vtk_use_sqlite_name_vtk_expects.patch`.

Was this PR tested with the default version of VTK?

Reply:
It was tested with vtk@:8 on Windows (no `@:9` support due to issues with `seacas`) (with various variants) and by whatever the Gitlab CI checks w.r.t. VTK options. 

Likely all of these patches can just be made "windows only".

Reply:
Let me know when you open a PR to fix this and I'll review.

Reply:
See: https://github.com/spack/spack/pull/42505